source/codegen/metadata/nidmm/attributes.py

```diff
@@ -30,7 +30,7 @@
     1050006: {
         'access': 'read-write',
         'channel_based': False,
-        'name': 'READ_COERCIONS',
+        'name': 'RECORD_COERCIONS',
```

Comment:
I don't see this attribute rename tracked in CHANGES.md. Is this also to match what's there in nidmm.h? Can you document this change.

Reply:
actually this was a typo when I first added this attribute to `attributes.py`, it is not really a name change. I have fixed it now. hence not recorded in CHANGES.md